glide-core/src/socket_listener.rs

```diff
@@ -350,6 +350,13 @@ fn get_command(request: &Command) -> Option<Cmd> {
         RequestType::HLen => Some(cmd("HLEN")),
         RequestType::Echo => Some(cmd("ECHO")),
         RequestType::ZPopMin => Some(cmd("ZPOPMIN")),
+        RequestType::XAck => Some(cmd("XACK")),
+        RequestType::XAdd => Some(cmd("XADD")),
+        RequestType::XReadGroup => Some(cmd("XREADGROUP")),
```

Comment:
maybe?
```suggestion
        RequestType::XReadGroup => Some(get_two_word_command("XREAD", "GROUP")),
```

Reply:
Should we apply this to other commands like `ZPOPMIN`?

Reply:
If I am not mistaken, `get_two_word_command` is for actual two-word commands , like [XGROUP CREATE](https://redis.io/commands/xgroup-create/)  that has a name with two words, unlike [XREADGROUP](https://redis.io/commands/xreadgroup/) that is a single word command:)

Reply:
🤦 
Security through obscurity